source/sass/components/article-page.scss

```diff
@@ -312,6 +312,41 @@
     @include set-text-size(16px, 24px);
     font-family: $font-family-sans-serif;
     font-weight: normal;
+
+    .rich-text {
+      * {
+        color: $gray-80;
+        font-size: 16px;
+      }
+
+      p {
+        max-width: initial;
+      }
+
+      b {
+        font-weight: bold;
+      }
+
+      a {
+        color: $dark-blue;
+      }
+
+      ol,
+      ul,
+      p {
+        margin-bottom: 12px;
+      }
+
+      ol > li::marker {
+        font-weight: normal;
+        font-size: 16px;
```

Comment:
I see that on line 234-236 we have

```scss
ol > li::marker {
    font-weight: bold;
  }
```
which sets ANY ordered list's marker on Article page to have bold weight. Can we just change that value from `bold` to `normal`? Or do we specifically want markers in **tables** on Article page to appear differently (that is, to have `normal` weight instead of `bold`)?

Reply:
Not sure how this got in here to be honest. I'm going to remove it.